libsession/src/main/java/org/session/libsession/messaging/jobs/GroupAvatarDownloadJob.kt

```diff
@@ -5,18 +5,38 @@ import org.session.libsession.messaging.open_groups.OpenGroupApi
 import org.session.libsession.messaging.utilities.Data
 import org.session.libsession.utilities.GroupUtil
 
-class GroupAvatarDownloadJob(val room: String, val server: String) : Job {
+class GroupAvatarDownloadJob(val server: String, val room: String, val imageId: String?) : Job {
```

Comment:
will `imageId` ever be null in the case that a download job gets called? Or does this also cover the case of handling removing the avatar after one has already been downloaded? I'm just wondering if we can safely make this non-null and simply not add it as a job if the parameter would have been null at the time we are creating it?

Reply:
Nah the `GroupAvatarDownloadJob` should never be created with a `null` `imageId`, the main reason I made it nullable was to avoid compatibility issues with backwards compatibility - I could probably just update the null check to be an `imageId.isEmpty()` check if you'd prefer

> Or does this also cover the case of handling removing the avatar after one has already been downloaded

It looks like this case isn't actually handled at all lol, I can make a commit to add in support for this as well

Reply:
haha yeah I realised the image deletion case wasn't handled after I checked where we call this from. I think we can add it later anyway if we need to, although now that we support the placeholder image it probably wouldn't look as bad removing the previous image to display. it probably makes sense to keep it nullable here I guess, we already have a check in execution :shrug: